tests/integration/test_scale_means.py

```diff
@@ -562,8 +547,8 @@ def test_univariate_with_hs_var_scale_means_row():
         "rows_dimension": {"insertions": {}},
     }
     strand = Cube(CR.ECON_BLAME_WITH_HS, transforms).partitions[0]
-    np.testing.assert_almost_equal(strand.var_scale_mean, [1.1363901131679894])
+    assert strand.scale_mean == pytest.approx(2.1735206)
```

Comment:
I see that considering that `var_scale_mean`  no longer exists (maybe for a reason but i don't know yet :D) ... BTW there the name of the test is wrong cause we're testing for scale means and not for its variance

Reply:
fixed the name. We wouldn't test a helper method in an integration test because implementation is subject to change and the integration test should still pass if it does. As far as I can tell, `_Strand.var_scale_means` was always a helper, even though it was named as interface.

In any case, `_ScaledCounts._scale_variance` is now tested in the unit tests for that measure.